src/prog_models/sim_result.py

```diff
@@ -36,6 +36,8 @@ def __init__(self, times: list = None, data: list = None, _copy=True):
 
     def __getitem__(self, item):
         """
+        .. versionadded:: 1.5.0
+        
```

Comment:
```suggestion
```
Implemented it in this release to add deprecation warning only- We don't need the message for it